### PR TITLE
[FIX] web_editor: add label in link_dialog from wysiwg

### PR DESCRIPTION
**Pull request by Goaman**
Open the link dialog from the wysiwyg with the label input.

Task-2704538

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Goaman** [comment]
@odoo/rd-security can you review this task ?

File: addons/web_editor/static/src/js/wysiwyg/widgets/link.js [hunk 1]
```diff
@@ -200,15 +205,7 @@ const Link = Widget.extend({
             this.$link[0].removeAttribute('target');
         }
         if (this._setLinkContent && (data.content !== this.data.originalText || data.url !== this.data.url)) {
```
Comment (mart-e) [inline addons/web_editor/static/src/js/wysiwyg/widgets/link.js:207]: we verify that the content has changed in `_setLinkWithData`, maybe we can simplify?

File: addons/web_editor/static/src/js/wysiwyg/widgets/link.js [hunk 2]
```diff
@@ -459,6 +456,26 @@ const Link = Widget.extend({
      * @param {boolean} [active]
      */
     _setSelectOption: function ($option, active) {},
+    /**
+     * Set the link from data.
```
Comment (mart-e) [inline addons/web_editor/static/src/js/wysiwyg/widgets/link.js:460]: "data" is not very descriptive

**Review by mart-e — COMMENTED** [review]
@robodoo override=ci/security
Reason: setting html is only done when the content hasn't changed

File: addons/web_editor/static/src/js/wysiwyg/widgets/link.js (hunk 2 above)
Comment (xmo-odoo) [reply addons/web_editor/static/src/js/wysiwyg/widgets/link.js:460]: Also "the link", that would usually be interpreted as the URL but here we're setting the label / content of the link element.

File: addons/web_editor/static/src/js/wysiwyg/widgets/link.js (hunk 1 above)
Comment (Goaman) [reply addons/web_editor/static/src/js/wysiwyg/widgets/link.js]: done

File: addons/web_editor/static/src/js/wysiwyg/widgets/link.js (hunk 2 above)
Comment (Goaman) [reply addons/web_editor/static/src/js/wysiwyg/widgets/link.js]: updated

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**